sys/dev/e1000/if_em.c

```diff
@@ -1907,6 +1909,10 @@ em_if_stop(if_ctx_t ctx)
 
 	INIT_DEBUGOUT("em_if_stop: begin");
 
+	/* I219 needs special flushing to avoid hangs */
+	if (sc->hw.mac.type >= e1000_pch_spt && sc->hw.mac.type < igb_mac_min)
```

Comment:
Looking here: https://git.dpdk.org/dpdk/tree/drivers/net/e1000/em_ethdev.c#n728, this check was updated to apply this workaround to cnp in addition to spt, but I see that the device ID list only goes up to (7) for DPDK, and not up to the current (23) that we're up to, so maybe this check doesn't need to change and we should apply it to everything spt and newer.